### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by Dank-del** [WarrenJones/vite-plugin-require-transform#34]
<img width="834" alt="Screenshot 2023-01-22 at 8 49 04 PM" src="https://user-images.githubusercontent.com/63096193/213923650-a9855d03-cef8-4702-8ed1-8686d7ac7bf9.png">

My package.json
```
{
  "name": "watch-cf",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "@consumet/extensions": "^1.3.0",
    "@silvermine/videojs-quality-selector": "^1.2.5",
    "react": "^17.0.2",
    "react-cookie": "^4.1.1",
    "react-dom": "^17.0.2",
    "react-helmet": "^6.1.0",
    "react-router-dom": "^6.6.1",
    "react-toastify": "^9.1.1",
    "string-strip-html": "^13.0.6",
    "swiper": "^8.4.5",
    "swr": "^2.0.0",
    "video.js": "^7.20.3",
    "vite": "^3.2.5"
  },
  "devDependencies": {
    "@vitejs/plugin-react": "^1.0.7",
    "autoprefixer": "^10.4.2",
    "daisyui": "^2.6.0",
    "postcss": "^8.4.7",
    "tailwindcss": "^3.0.23",
    "vite-plugin-require-transform": "^1.0.6"
  }
}

```

**Comment by ryanlntn** [comment]
This seem to be related to [this change](https://github.com/WarrenJones/vite-plugin-require-transform/commit/7b204c1482a99a65e8354b5d2c18f1361ae574d6#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L7). Reverting to `1.0.4` worked for me.

**Comment by WarrenJones** [comment]
> <img alt="Screenshot 2023-01-22 at 8 49 04 PM" width="834" src="https://user-images.githubusercontent.com/63096193/213923650-a9855d03-cef8-4702-8ed1-8686d7ac7bf9.png">
> 
> My package.json
> 
> ```
> {
>   "name": "watch-cf",
>   "private": true,
>   "version": "0.0.0",
>   "type": "module",
>   "scripts": {
>     "dev": "vite",
>     "build": "vite build",
>     "preview": "vite preview"
>   },
>   "dependencies": {
>     "@consumet/extensions": "^1.3.0",
>     "@silvermine/videojs-quality-selector": "^1.2.5",
>     "react": "^17.0.2",
>     "react-cookie": "^4.1.1",
>     "react-dom": "^17.0.2",
>     "react-helmet": "^6.1.0",
>     "react-router-dom": "^6.6.1",
>     "react-toastify": "^9.1.1",
>     "string-strip-html": "^13.0.6",
>     "swiper": "^8.4.5",
>     "swr": "^2.0.0",
>     "video.js": "^7.20.3",
>     "vite": "^3.2.5"
>   },
>   "devDependencies": {
>     "@vitejs/plugin-react": "^1.0.7",
>     "autoprefixer": "^10.4.2",
>     "daisyui": "^2.6.0",
>     "postcss": "^8.4.7",
>     "tailwindcss": "^3.0.23",
>     "vite-plugin-require-transform": "^1.0.6"
>   }
> }
> ```

Thanks for reporting,it's been fixed in version 1.0.8,you can check it out.Actually now I'm still not familiar with commonjs and esm but there will be a perfect landing not far in the future.
